tests/unit/test_storage.py

```diff
@@ -30,19 +30,20 @@ def tearDown(self):
     def test_location(self):
         """Test file location."""
         result = self._store.location('12')
-        self.assertEqual(os.path.join(self.outputdir, '000', '000', '000', '000', '000', '000', '12'), result)
+        self.assertEqual(os.path.join(self.outputdir + '/lega/', '000', '000', '000', '000', '000', '000', '12'), result)
```

Comment:
Let os.path.join add the slashes.
You can update to: ```self.outputdir, 'lega', '000', '000', '000', '000', '000', '000', '12'```.
It's also a teeny-weeny more efficient.

Reply:
true, updated